sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -155,6 +155,29 @@ template <> inline void print<>(PiPlatform val) {
   std::cout << "pi_platform : " << val << std::endl;
 }
 
+template <> inline void print<>(PiEvent val) {
+  std::cout << "pi_event : " << val << std::endl;
+}
+
+template <> inline void print<>(PiMem val) {
```

Comment:
Can we move print function to source directory to reduce number of public APIs?
@alexbatashev  FYI

Reply:
Done.